core-backend/common/src/funcs.rs

```diff
@@ -93,6 +93,20 @@ pub fn gas_available<E: Ext>(ext: LaterExt<E>) -> impl Fn() -> i64 {
     move || ext.with(|ext: &mut E| ext.gas_available()).unwrap_or(0) as i64
 }
 
+pub fn kill<E: Ext>(
```

Comment:
what do you think about changing name to `interrupt` or `stop`?
As far as I remember we plan to make a feature of `selfdestruct`, when program can be deleted (erase state, delete address and e.t.c.). I think `kill` and `selfdestruct` names point to the same semantics, although planned to differ.

Reply:
To tell the truth I'm not insisting on this name, so probably it will be changed. For example, `terminate` is ok too.

What about the syscall behavior: we discussed the task last week. When the program would call `kill` its state should be removed, however the hash of the state and all waiting messages to the program should be kept (and program's id too as I guess). This is for the next feature that allows users to restore the killed program.

Reply:
I would go with `exit` as in `std::process:exit`